packages/block-library/src/embed/edit.native.js

```diff
@@ -45,8 +45,6 @@ const EmbedEdit = ( props ) => {
 	const { icon, title } =
 		getEmbedInfoByProvider( providerNameSlug ) || defaultEmbedInfo;
 
-	const [ url, setURL ] = useState( attributesUrl );
-
```

Comment:
I noticed that for the mobile version we don't really need to keep the URL in the internal state, as we were only modifying it once it changed, and that's already being handled with the URL block's attribute.